packages/web-frontend/cypress/support/helpers.js

```diff
@@ -3,26 +3,6 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-/* eslint-disable max-classes-per-file */
-
-import { CONFIG_ROOT } from '../constants';
-
-export const getConfigPath = fileName => `${CONFIG_ROOT}/${fileName}.json`;
-
-const getConfigExamplePath = fileName => `${CONFIG_ROOT}/${fileName}.example.jsonc`;
-
-export class EmptyConfigError extends Error {
-  constructor(fileName) {
-    const path = getConfigPath(fileName);
-    const examplePath = getConfigExamplePath(fileName);
-    const generationCommand = '`yarn workspace @tupaia/web-frontend cypress:generate-config`';
-    const message = `Test config error: ${path} contains no test data. Either use ${generationCommand} to populate this file, or populate it with manual data. See ${examplePath} for an example`;
```

Comment:
Let's keep things simple - the users of our code should know (or ask) what to do if they get an "empty config" error. Plus, I added config documentation in this PR.